src/ringct/rctTypes.h

```diff
@@ -106,6 +107,8 @@ namespace rct {
         key L;
         key R;
         key ki;
+
+        ~multisig_kLRki() { memwipe(&k, sizeof(k)); }
```

Comment:
Does this mean when a kLRki struct is destroyed, it also memwipes the addresses corresponding to k? Seems quite useful

Reply:
Yes.